docs/_content/overige/technisch/zds-vs-zgw.md

```diff
@@ -7,27 +7,22 @@ De Zaak- en Documentservices (ZDS) wijken af van de Zaakgericht werken (ZGW)
 APIs, maar er is ook overlap. De belangrijkste verschillen en overeenkomsten op
 een rij:
 
-1. De ZGW APIs zijn in REST/JSON terwijl ZDS volgens SOAP/XML werken.
-2. De ZGW APIs zijn veel flexibeler dan ZDS. Dit is nodig omdat er geen directe
-   acties meer op de database mag worden gedaan door bijvoorbeeld een 
-   Zaaksysteem. Alle acties moeten via de ZGW APIs verlopen.
-3. De ZGW APIs kunnen alles wat ZDS ook kan.
-4. De ZGW APIs zijn opgedeeld. Er is niet meer 1 specificatie of endpoint om
-   alle operaties, zoals deze in ZDS bestonden, uit te voeren of data op te
-   vragen.
-5. De ZGW APIs zullen sneller opvolgende versies krijgen. Hierbij is het niet
-   ondenkbaar dat de Zaken API op versie 2 zit, terwijl de Besluiten API al op
-   versie 4 zit. Het is daarom ook niet zo dat ZGW een versie nummer krijgt.
-   Immers is ZGW alleen maar de verzamelnaam van alle Zaakgericht werken APIs. 
-6. De ZGW APIs worden aangetoond middels een referentie implementatie om de
-   theoretische specificaties te toetsen aan de praktijk, migratie problemen
-   te detecteren en om het vereiste gedrag te definieren daar waar de 
-   specificatie ruimte laat voor interpretaties.
-7. De ZGW APIs zijn ontwikkeld vanuit developer-first oogpunt waardoor
-   koppelingen sneller gemaakt kunnen worden.
-8. De ZGW APIs kennen alleen synchrone operaties.
-9. De ZGW APIs worden net als ZDS beheerd en onderhouden door VNG, zij het in
-   een andere vorm.
+[Zaak-Documentservices](https://www.gemmaonline.nl/index.php/Zaak-_en_Documentservices) | [ZGW APIs](https://ref.tst.vng.cloud/standaard/index)\*
+--- | ---
+SOAP/XML gebaseerd	                                                                        | REST/JSON gebaseerd
+Gebaseerd op het informatiemodel RGBZ1	                                                    | Gebaseerd op informatiemodel RGBZ2 waar van toepassing
+Verdere invulling/uitwerking van de basisberichten uit StUF ZKN	                            | Aparte calls voor aparte resources
+Gebaseerd op uitwisselen van informatie, gestructureerd volgens het informatiemodel [RGBZ 1](https://www.gemmaonline.nl/index.php/Informatiemodel_Zaken_(RGBZ))	| Gebaseerd op operaties op resources die leven in het referentiecomponent
```

Comment:
Beetje scheve vergelijking?

Beide zijn gebaseerd op uitwisselen van informatie, maar.. het is StUF-ZKN vs OASen van de ZGW API's

Ook klinkt "...leven in het referentiecomponent" vreemd, want het referentiecomponent is alleen maar een hulpmiddel bij de standaard.

Reply:
Ik zie toch wel verschil: in StUF berichten kunnen gegevens opgenomen zijn die niet of niet volledig in het Zaaksysteem opgeslagen worden. Bovendien worden in StUF berichten alle soorten gegevens in 1 bericht (bv om een zaak aan te maken) gestopt. Terwijl bij de ZGW API's de resources apart opgevoerd worden en alleen een verwijzing naar de overige resources opgenomen wordt in de zaak.

Reply:
Leven in referentiecomponent moet eigenlijk zijn leven in een registratie.

Reply:
> Ik zie toch wel verschil

Er is ook een verschil, maar ik vind het scheef lopen dat je aan de ene kant "structuur RGBZ" noemt, en aan de andere kant niets over structuur zegt.

Reply:
Wat ik wil zeggen is dat StUF ZKN en de daarvan afgeleide koppelvlakken de structuur van het RGBZ volgen. Bij het opzetten van de API's is deze, "verplichte", structuur losgelaten en gekeken welke resources en attributen van toepassing zijn op de user stories.
Uiteindelijk is er een model uitgekomen wat voor een heel groot deel overeenkomt met het RGBZ maar dat model is niet vantevoren opgelegd, zoals dat bij StUF ZKN wel zo is.
Hopelijk is een beetje duidelijk wat ik bedoel, ik ben het vehaal toch aan het omschrijven, hopelijk komt het dan beter uit de verf.

Reply:
Ik vind de nieuwe formulering prima :+1: